src/main/java/com/blog/dao/impl/TagDaoRestClientImpl.java

```diff
@@ -24,14 +25,28 @@
 public class TagDaoRestClientImpl extends RestClientAbstract implements TagDao {
 
     private static final Logger LOGGER = LogManager.getLogger();
+    private static final String SLASH = "/";
 
     @Autowired
-    public TagDaoRestClientImpl(RestTemplate restTemplate, HttpHeaders headers) {
+    public TagDaoRestClientImpl(RestTemplate restTemplate, HttpHeaders headers, ObjectMapper jsonConverter) {
         this.restTemplate = restTemplate;
         this.headers = headers;
+        this.jsonConverter = jsonConverter;
         endpoint = "tags";
```

Comment:
It's not an error. I personally prefer to use abstract method `getEndpoint()` instead of protected variables because I can forget to set this value in constructor, but strict typing will force implement method